src/cascadia/TerminalConnection/ConptyConnection.cpp

```diff
@@ -215,9 +215,7 @@ namespace winrt::Microsoft::Terminal::TerminalConnection::implementation
         _piClient.hProcess = hClientProcess;
 
         _startupInfo.title = winrt::hstring{ startupInfo.pszTitle, SysStringLen(startupInfo.pszTitle) };
-        SysFreeString(startupInfo.pszTitle);
         _startupInfo.iconPath = winrt::hstring{ startupInfo.pszIconPath, SysStringLen(startupInfo.pszIconPath) };
-        SysFreeString(startupInfo.pszIconPath);
```

Comment:
We should pass `startupInfo` as `const&` to indicate that the value is borrowed.